config/chime_science_run_recv.yaml

```diff
@@ -47,6 +47,9 @@ vis_buffer:
   visbuf_10s_flags:
     kotekan_buffer: vis
     buffer_depth: 384  # Before slow stage
+  visbuf_10s_flags_delay:
+    kotekan_buffer: vis
+    buffer_depth: 384  # Before slow stage
```

Comment:
It's not clear we need this many frames in the output buffer?   Also we shouldn't commit this change until we get more memory in `recv2` or this will just fail to run.